DigitalLearningSolutions.Web/Views/Home/LearningContent.cshtml

```diff
@@ -0,0 +1,75 @@
+
+    <vc:contents-list mini-hub-navigation-model="@Model.MiniHubNavigationModel" />
+
+    <div class="content-area">
+
+      <h2 class="content-area__heading">
+        IT Skills Pathway
+      </h2>
+
+      <div class="learning-content-item__details-container">
+        <div class="content-area__image-box">
+          <img class="content-area__image" src="@Url.Content("~/images/itSkillsPathwayLogo.png")" alt="It Skills Pathway logo"/>
+        </div>
+        <div class="learning-content-item__text-container">
+          <p class="learning-content-item__text">Quality assured Digital Skills and Microsoft Office learning and assessment content. From basic digital literacy and skills through to advanced Microsoft Excel and Word training.</p>
+          <a class="learning-content-item__text" asp-controller="LearningContent" asp-action="ItSkillsPathway">See more</a>
+        </div>
+      </div>
+
+    </div>
+
+    <div class="content-area">
+
+      <h2 class="content-area__heading">
+        Reasonable Adjustment Flag
+      </h2>
+
+      <div class="learning-content-item__details-container">
+        <div class="content-area__image-box">
+          <img class="content-area__image" src="@Url.Content("~/images/reasonableAdjustmentFlagLogo.png")" alt="Reasonable Adjustment Flag logo"/>
+        </div>
+        <div class="learning-content-item__text-container">
+          <p class="learning-content-item__text">The reasonable adjustment flag enables health and care professionals to record, share and view patients’ reasonable adjustments across the NHS. This enables staff and services to carry out this anticipatory duty wherever the patient is treated.</p>
+          <a class="learning-content-item__text" asp-controller="LearningContent" asp-action="ReasonableAdjustmentFlag">See more</a>
+        </div>
+      </div>
+
+    </div>
+
+    <div class="content-area">
+
+      <h2 class="content-area__heading">
+        Terminology and Classifications Delivery Service
+      </h2>
+
+      <div class="learning-content-item__details-container">
+        <div class="content-area__image-box">
+          <img class="content-area__image" src="@Url.Content("~/images/nhsDigitalLogo.png")" alt="NHS Digital logo"/>
+        </div>
+        <div class="learning-content-item__text-container">
+          <p class="learning-content-item__text">Learning resources relating to Clinical and Pharmacy Terminology and Clinical Classifications products, including ICD, OPCS, SNOMED CT and dm+d. These resources will help you to understand and make best use of these products.</p>
+          <a class="learning-content-item__text" asp-controller="LearningContent" asp-action="TerminologyAndClassificationsDeliveryService">See more</a>
+        </div>
+      </div>
+
+    </div>
+
+    @await Component.InvokeAsync(typeof(PaginationViewComponent), new { miniHubNavigationModel = Model.MiniHubNavigationModel })

```

Comment:
Can we use the tag helper syntax here too?

Reply:
D'oh